### DISCUSSION TITLE
How to install Docker Desktop on CentOS 9?

### DISCUSSION THREAD

**Discussion by Delicious-Bacon** [moby/moby#44105] [Q&A] [answered]
I run Rocky Linux 9 and I don't know how to install Docker Desktop.

Here's what I run:

```
$ sudo dnf install docker-desktop-4.11.0-x86_64.rpm 
Rocky Linux 9 - BaseOS                          4.4 kB/s | 3.6 kB     00:00    
Rocky Linux 9 - AppStream                       7.1 kB/s | 3.6 kB     00:00    
Rocky Linux 9 - AppStream                       3.6 MB/s | 6.0 MB     00:01    
Rocky Linux 9 - CRB                             8.9 kB/s | 3.6 kB     00:00    
Error: 
 Problem: conflicting requests
  - nothing provides qemu-system-x86 >= 5.2.0 needed by docker-desktop-4.11.0-83626.x86_64
  - nothing provides pass needed by docker-desktop-4.11.0-83626.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

It looks like I don't have `qemu-system-x86` and `pass`.

I do have `qemu-kvm`:
```
$ sudo dnf install qemu-kvm
Last metadata expiration check: 0:12:05 ago on Wed 07 Sep 2022 09:44:13 AM CEST.
Package qemu-kvm-17:6.2.0-11.el9_0.3.x86_64 is already installed.
Dependencies resolved.
Nothing to do.
Complete!
```

I don't understand how to provide the `qemu-system-x86` through it.

As for `pass`, I don't even know where to begin looking for that because of such a generic name.

**Comment by thaJeztah** [comment, marked as answer]
This looks specific to Docker Desktop for Linux, which is not maintained in this project; could you open a ticket in the Docker Desktop for Linux issue tracker instead? https://github.com/docker/desktop-linux/issues

**Reply by Delicious-Bacon** [reply]
I have opened it as an [issue there](https://github.com/docker/desktop-linux/issues/71#issue-1364775829) now.

I thought that repo was for cases when Docker Desktop already runs, since it asks for the Diagnostics ID when creating an issue. :shrug:

**Comment by wmc789** [comment]
got the same error running Alma 8.6


Last metadata expiration check: 0:05:11 ago on Fri 30 Sep 2022 08:55:21 AM CEST.
Error: 
 Problem: conflicting requests
  - nothing provides pass needed by docker-desktop-4.12.0-85629.x86_64
  - nothing provides qemu-system-x86 >= 5.2.0 needed by docker-desktop-4.12.0-85629.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
